tests/switchmap_/dashboard/table/test_device.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import unittest
+import os
+import sys
+from unittest.mock import patch
+
+# Add the project root directory to sys.path
+ROOT_DIR = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "../../../../")
```

Comment:
1. This is going to fail in the workflows.
2. The script needs to be able to run from the root directory of the repository
3. Follow the other test  examples in the test directory.